### PR TITLE
Changing tag format in config file

### PR DESCRIPTION
**Pull request by ch40s3k**

### CODE REVIEW

**Comment by h00dy** [comment]
:+1:

File: src/diamond/handler/kairosdb.py [hunk 1]
```diff
@@ -2,11 +2,26 @@
 Send metrics to a [KairosDB](http://kairosdb.github.io/) using telnet or rest
 method.
 
+Add the following configuration to diamond.conf:
+
+[[KairosDBHandler]]
+host = localhost
+port = 4242
+
+Optionaly if you like to use Tags, you should add field [[[tags]]] in collector config.
```
Comment (deejay1) [inline src/diamond/handler/kairosdb.py]: s/field/section/

Also lower case "Tags"

File: src/diamond/handler/kairosdb.py [hunk 2]
```diff
@@ -2,11 +2,26 @@
 Send metrics to a [KairosDB](http://kairosdb.github.io/) using telnet or rest
```
Comment (deejay1) [inline src/diamond/handler/kairosdb.py]: REST

File: src/diamond/handler/kairosdb.py (hunk 1 above)
Comment (ch40s3k) [inline src/diamond/handler/kairosdb.py]: Done

**Comment by deejay1** [comment]
:+1:

**Pull request merged**